package.json

```diff
@@ -9,20 +9,22 @@
     "vendor:subql:install": "cd vendor && yarn workspace @subql/node-cosmos install",
     "vendor:cosmjs:build": "cd vendor && yarn workspace cosmjs-monorepo-root run build",
     "vendor:subql:build": "cd vendor && yarn workspace @subql/node-cosmos run build",
-    "vendor:clean": "cd vendor && rm -rf node_modules && find ./ -type d -name 'node_modules' -exec sh -c 'rm -rf \"$0\"/*' {} \\;",
-    "docker:build": "docker-compose build",
-    "docker:build:no-cache": "docker-compose build --no-cache",
-    "build": "subql codegen &&  sed -i 's/\\[key: bigint\\]/\\[key: number\\]/g' ./src/types/proto-interfaces/poktroll/application/application.ts && subql build",
+    "vendor:clean": "cd vendor && find . -name node_modules | xargs rm -rf && find . -name dist | xargs rm -rf",
+    "docker:build": "docker compose build",
+    "docker:build:no-cache": "docker compose build --no-cache",
+    "build": "subql codegen && sed -i 's/\\[key: bigint\\]/\\[key: number\\]/g' ./src/types/proto-interfaces/poktroll/application/application.ts && subql build",
     "build:develop": "NODE_ENV=develop subql codegen &&  sed -i 's/\\[key: bigint\\]/\\[key: number\\]/g' ./src/types/proto-interfaces/poktroll/application/application.ts && NODE_ENV=develop subql build",
     "codegen": "subql codegen",
-    "docker:start": "docker-compose pull && docker-compose up --remove-orphans",
-    "docker:stop": "docker-compose down",
+    "docker:start": "docker compose pull && docker compose up --remove-orphans",
+    "docker:stop": "docker compose down",
     "docker:clean": "yarn docker:stop && docker volume rm pocketdex_postgres_data",
     "docker:tunnel": "echo 'if public key auth fails the password is `proxypass`.' && ssh -o StrictHostKeyChecking=no -N -R 26657:localhost:26657 proxyuser@localhost -p 2222",
-    "dev": "subql codegen && subql build && docker-compose pull && docker-compose up --remove-orphans",
+    "docker:dev:local": "docker compose up -d postgres pgadmin graphql-engine",
+    "dev:docker": "subql codegen && subql build && docker compose pull && docker compose up --remove-orphans",
+    "dev:local": "NODE_ENV=develop node vendor/subql-cosmos/packages/node/bin/run -f=./ --db-schema=app --workers=1 --batch-size=10 --unfinalized-blocks=true",
```

Comment:
There are some things here I think need reformulating but don't have the energy to articulate tonight. I will follow up in the morning.

Reply:
100% we need to go over these scripts once we reach a more stable product because otherwise they are a bit chaotic

Reply:
I'm fine merging this as-is so long as the README is up-to-date. Deal? :handshake: 

_(Bonus points for opening an issue to clean up the scripts.)_

Reply:
> Do not leave for tomorrow what you can do today

So I simplified the scripts, cleaned up things a bit, and added `nodemon` for hot-reload. Is not perfect, there is still room for improvement around docker builds but you should not need to build it if do not change vendor or package.json